app/static/src/app/components/options/AdvancedSettings.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div class="container">
+        <div class="row my-2" v-for="(config, option) in advancedSettings" :key="option">
+            <div class="col-5">
+            <label class="col-form-label">{{ option }}</label>
+            </div>
+            <div class="col-6">
+                <standard-form-input v-if="config.standardForm"
+                                     :value="config.val"
+                                     :placeholder="config.defaults"
+                                     @update="(n) => updateOption(n, option)"/>
+                <numeric-input v-else
+                               :value="config.val"
+                               :placeholder="config.defaults"
+                               @update="(n) => updateOption(n, option)"/>
+            </div>
+        </div>
+    </div>
+</template>
+<script lang="ts">
+import { computed, defineComponent } from "vue";
+import { useStore } from "vuex";
+import NumericInput from "./NumericInput.vue";
+import StandardFormInput from "./StandardFormInput.vue";
+import { AdvancedSettings, AdvancedOptions } from "../../store/run/state";
+import { RunMutation } from "../../store/run/mutations";
+
+export default defineComponent({
+    components: {
```

Comment:
I think that a "reset" button like on the Model Parameters component might be wise. Could be another ticket though

Reply:
yep agreed! created a ticket for this: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4478/Reset-button-for-advanced-settings